var/spack/repos/builtin/packages/py-elephant/package.py

```diff
@@ -28,40 +31,44 @@ class PyElephant(PythonPackage):
     version("0.3.0", sha256="747251ccfb5820bdead6391411b5faf205b4ddf3ababaefe865f50b16540cfef")
 
     variant("docs", default=False, description="Install documentation dependencies")
-    variant("pandas", default=False, description="Build with pandas", when="@0.3.0:0.4.1")
     variant(
         "extras", default=False, description="Build with extras for GPFA, ASSET", when="@0.6.4:"
     )
 
     depends_on("py-setuptools", type="build")
-    depends_on("python@3.7:", type=("build", "run"), when="@0.11.0:")
+
+    depends_on("python@3.8:", type=("build", "run"), when="@0.12.0:")
+
+    depends_on("py-neo@0.10.0:", type=("build", "run"), when="@0.11.0:")
+    depends_on("py-neo@0.9.0", type=("build", "run"), when="@0.9.0:0.10.0")
+    depends_on("py-neo@0.8.0", type=("build", "run"), when="@0.6.4:0.8.0")
     depends_on("py-neo@0.3.4:", type=("build", "run"), when="@0.3.0:0.4.1")
+    depends_on("py-numpy@1.19.5:", type=("build", "run"), when="@0.12.0:")
+    depends_on("py-numpy@1.18.1:1.23.5", type=("build", "run"), when="@0.6.4:0.11.2")
     depends_on("py-numpy@1.8.2:", type=("build", "run"), when="@0.3.0:0.4.1")
-    depends_on("py-quantities@0.10.1:", type=("build", "run"), when="@0.3.0:0.4.1")
+    depends_on("py-quantities@0.14.1:", type=("build", "run"), when="@0.12.0:")
+    depends_on("py-quantities@0.12.1:0.13.0", type=("build", "run"), when="@0.6.4:0.11.2")
+    depends_on("py-quantities@0.10.1:0.13.0", type=("build", "run"), when="@0.3.0:0.4.1")
```

Comment:
Where do these upper bounds come from?

Reply:
A bug in quantities 0.14.0 caused some errors in the calculations, so we had to avoid using that version. 
The bug was fixed in 0.14.1 and I just realized that 0.14.1 also works for older elephant versions.

Thanks for pointing me to this.

I updated the requirements accordingly.